packages/Webkul/Admin/src/DataGrids/CategoryDataGrid.php

```diff
@@ -3,22 +3,46 @@
 namespace Webkul\Admin\DataGrids;
 
 use Illuminate\Support\Facades\DB;
+use Webkul\Core\Models\Locale;
 use Webkul\Ui\DataGrid\DataGrid;
+use Webkul\Ui\DataGrid\Traits\ProvideDataGridPlus;
 
 class CategoryDataGrid extends DataGrid
 {
+    use ProvideDataGridPlus;
```

Comment:
This thing is not needed as you already using the `render()` method in the blade file. So this should be default as previous.